src/config/index.ts

```diff
@@ -114,12 +114,28 @@ export const ENTITY_CHANGE_LEGAL_FORM = ENTITY_URL + LEGAL_FORM;
 export const ENTITY_CHANGE_GOVERNING_LAW = ENTITY_URL + LAW_GOVERNED;
 export const ENTITY_CHANGE_PUBLIC_REGISTER = ENTITY_URL + "#is_on_register_in_country_formed_in";
 
+const IDENTITY_DATE = "#identity_date-day";
+const IDENTITY_NAME = "#name";
```

Comment:
Call it name, it is needed in other fields

Reply:
Updated .